### PR TITLE
Modified the Simplified Chinese [zh-CN] and added Traditional Chinese [zh-TW] translation

### PR DESCRIPTION
**Pull request by eason329**
Hi, I've modified the Simplified Chinese [**zh-CN**] and added Traditional Chinese [**zh-TW**] translation to the resources folder, to include in the predefined translations.

**If only ONE zh translation can be added, use the Simplified Chinese one by changing the language tag from `zh-Hans` to `zh`, or else the original `zh` translation will be replaced.**

Note: I made a mistake by including "# 2" (which is an closed issue) in the title of some commit changes, please ignore it.
(EDIT): I don't have enough ability to proofread the translation so the proficiency of these translations are Intermediate.
**(EDIT 2): The language tag I previously used does not meet the requirement in readme, use `zh-CN` for Simplified Chinese and `zh-TW` for Traditional Chinese instead. Sorry for making another mistake**

### CODE REVIEW

File: src/resources/Comms_zh-CN.properties [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+secondary=§f
+highlightBefore=§4
+highlightAfter=§2
+strong=§l
+italic=§o
+strike=§m
+error=§c
+# Display
+prefix={primaryDark}[{primary}{0}{primaryDark}]{reset} {1}
+print_line= {0}＜{1,number}＞{reset} {2}
```
Comment (Deltik) [inline src/resources/Comms_zh-CN.properties]: The prepended space here will be ignored.  If you intended for there to be a literal space, it needs to be changed to this:
```suggestion
print_line=\\ {0}＜{1,number}＞{reset} {2}
```

File: src/resources/Comms_zh-Hant.properties [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+secondary=§f
+highlightBefore=§4
+highlightAfter=§2
+strong=§l
+italic=§o
+strike=§m
+error=§c
+# Display
+prefix={primaryDark}[{primary}{0}{primaryDark}]{reset} {1}
+print_line= {0}＜{1,number}＞{reset} {2}
```
Comment (Deltik) [inline src/resources/Comms_zh-Hant.properties]: The prepended space here will be ignored.  If you intended for there to be a literal space, it needs to be changed to this:
```suggestion
print_line=\\ {0}＜{1,number}＞{reset} {2}
```

File: src/resources/Comms_zh-CN.properties [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+you_cannot_use={error}你没有权限使用 {primaryLight}{0}{error}。
+usage_section={secondary}{strong}使用方法：
+print_subcommand_usage={primary}/{0} {primaryLight}{1} {primaryDark}{2}
+online_documentation={secondary}{strong}在线帮助：{reset}{0}
+online_documentation_url=https://git.io/SignEdit-README
+sign_did_not_change={primary}告示牌没有变更。
+before_section={primary}{strong}变更前：
+after_section={primary}{strong}变更后：
+right_click_sign_to_apply_action={primary}右键点击要编辑的告示牌
+right_click_sign_to_apply_action_hint= {italic}右键点击要编辑的告示牌
```
Comment (Deltik) [inline src/resources/Comms_zh-CN.properties]: The prepended space here will be ignored.  If you intended for there to be a literal space, it needs to be changed to this:
```suggestion
right_click_sign_to_apply_action_hint=\\ {italic}右键点击要编辑的告示牌
```

File: src/resources/Comms_zh-Hant.properties [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+you_cannot_use={error}你沒有權限使用 {primaryLight}{0}{error}。
+usage_section={secondary}{strong}使用方法：
+print_subcommand_usage={primary}/{0} {primaryLight}{1} {primaryDark}{2}
+online_documentation={secondary}{strong}線上幫助：{reset}{0}
+online_documentation_url=https://git.io/SignEdit-README
+sign_did_not_change={primary}告示牌沒有變更。
+before_section={primary}{strong}變更前：
+after_section={primary}{strong}變更後：
+right_click_sign_to_apply_action={primary}右鍵點擊要編輯的告示牌
+right_click_sign_to_apply_action_hint= {italic}右鍵點擊要編輯的告示牌
```
Comment (Deltik) [inline src/resources/Comms_zh-Hant.properties]: The prepended space here will be ignored.  If you intended for there to be a literal space, it needs to be changed to this:
```suggestion
right_click_sign_to_apply_action_hint=\\ {italic}右鍵點擊要編輯的告示牌
```

**Review by Deltik — CHANGES_REQUESTED** [review]
Did you intend for there to be literal spaces here?  If so, those spaces have to be escaped with `\\`.

**Comment by eason329** [comment]
> Did you intend for there to be literal spaces here? If so, those spaces have to be escaped with `\\`.

_(This reply applies to all change request)_
Yes I was intended for it. However, when I test the translation file in my server, the double backslash here will cause a issue as shown below:


But the issue can be solved by using only one backslash as shown below (idk why):
螢幕擷取畫面 (140)

btw I will still commit your suggestion for now while I will continue to test. Thanks for your suggestion.
Note: It's now 1am already for me.

**Comment by Deltik** [comment]
Thank you for the contribution!  It is live now as [SignEdit v1.11.4](https://github.com/Deltik/SignEdit/releases/tag/v1.11.4)!

----

The double backslashes didn't seem to cause any literal backslashes in my build.

**Comment by AobiYT** [comment]
Thanks, but it seems you forgot the zh-TW one.

**Comment by eason329** [comment]
Nice (maybe that issue is my problem)

**Pull request merged**